packages/tasks/src/tasks/image-feature-extraction/about.md

```diff
@@ -0,0 +1,24 @@
+## Use Cases
+
+### Transfer Learning
+
+Models trained on a specific dataset can learn very shallow features about the data. For instance, a model trained on car classification dataset learns to recognize edges and curves on a very high level and car-specific features on a low level. This information can be transferred to a new model that is going to be trained on classifying trucks. This way of extracting features and transferring to a model is called feature extraction. One can pass their dataset through a feature extraction pipeline and feed the result to a classifier directly.
+
+### Similarity
+
+Features extracted from models contain semantically meaningful information about the world. These features can be used to detect the similarity between two images. Assume there are two images, a photo of a stray cat in a street setting, and a photo of a cat at home. These images both contain cats, and the features will contain the information that there's a cat in the image. Thus, comparing the features of a stray cat photo to the features of a domestic cat photo will result in higher similarity compared to any other image that doesn't contain any cats.
+
+## Inference
+
+```python
+import torch
+from transformers import pipeline
+
+DEVICE = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Let's aim to be consistent with how we set device across the task pages. Some task pages don't use `cuda`, some set `cuda` directly, some do this approach. Let's make them uniform

Reply:
I've observed this too. My preference would be to remove all device placement code in the task pages, for simplicity, and to focus on the task itself. However, it's also useful to learn these details. Perhaps we could link to some relevan transformers docs where appropriate? Example: https://huggingface.co/docs/transformers/pipeline_tutorial

Reply:
Sure I'll open a PR